keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -828,6 +869,12 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+Inspect the `supplementalGroupsPolicy` fields in Pods. You can check if the following `jq` command prints non-zero number:
```

Comment:
If you have 50,000 clusters this is not helpful. Is there a metric we can use, can kube state metrics help here?

Reply:
Yeah, kube state metrics can help. Let me update this section.